src/frontend/context/SpråkContext.tsx

```diff
@@ -28,5 +32,15 @@ export const [SpråkProvider, useSpråk] = createUseContext(() => {
         setCookie(dekoratorLanguageCookieName, language.locale);
     });
 
-    return { valgtLocale };
-});
+    return <SpråkContext.Provider value={{ valgtLocale }}>{props.children}</SpråkContext.Provider>;
+}
+
+export function useSpråk() {
```

Comment:
Er det bedre å ha suffixen `Context` for å gjøre tydelig at man tar i bruk en `Context`? Da blir det kanskje litt tydeligere at man må ha en provider, samt at endringer vil forårsake rerenders? Ser det er litt forskjellig hva som er brukt i prosjektene, noen hooks har det mens andre ikke har det

```suggestion
export function useSpråkContext() {
```

Reply:
Jeg kan godt endte navnet ja, `useSpråkContext` er jo litt mer beskrivende enn før. 😊 Det har jo ingenting å si akkurat her der vi ikke setter noe i contexten fra komponenter, men på andre contexter der vi eksponerer "set"-metoder er det jo veldig verdifull informasjon. Jeg fikser! 🙌